src_compressible/flux.cpp

```diff
@@ -281,8 +409,15 @@ void calculateFlux(const MultiFab& cons_in, const MultiFab& prim_in,
                     Real soret = 0.;
 
                     for (int ns=0; ns<nspecies; ++ns) {
-                        soret = soret + (hk[ns] + Runiv*meanT/molmass[ns]
-                                         *0.5*(chi(i-1,j,k,ns)+chi(i,j,k,ns)))*wiener[5+ns];
+                        Real soret_s;
+                        soret_s = (hk[ns] + Runiv*meanT/molmass[ns]*0.5*(chi(i-1,j,k,ns)+chi(i,j,k,ns)))*wiener[5+ns];
+                        if ((i == 0) and (bc_mass_lo[0] != 3) and (bc_mass_lo[0] != -1)) { // ignore for reservoirs and periodic BC
+                            soret_s = (hk[ns] + Runiv*meanT/molmass[ns]*0.5*chi(i-1,j,k,ns))*wiener[5+ns];
```

Comment:
Don't want 0.5 here.  Ditto for all the soret_s terms on the boundary.

Reply:
Good catch - thanks.